src/data/fullness.txt

```diff
@@ -129,7 +129,7 @@ bucket of honey	1	4	good	3	0	0	0	50 Sugar Rush
 bunch of square grapes	2	7	awesome	6-7	1-5	5-20	1-5
 C.H.U.M. chum	3	1	crappy	1	0	0	0	20 Majorly Poisoned
 cactus fruit	2	7	awesome	6-7	6-20	1-5	1-5	lose 8-10 HP
-Calzone of Legend	2	5	EPIC	0	0	0	0	Unspaded, 100 In the 'zone zone! (+50 Spell Damage, +25 Mys, +300% Mys, +100% Max HP/MP), PIZZA
+Calzone of Legend	2	5	EPIC	19-22	0	0	0	Unspaded, 100 In the 'zone zone! (+50 Spell Damage, +25 Mys, +300% Mys, +100% Max HP/MP), PIZZA
```

Comment:
Didn't you say 19-21?

Reply:
Luckily for me, it seems my error was in writing the PR description. We definitely got reports of 24 with pizza lover.

Reply:
OK.